src/components/Delete-Task.js

```diff
@@ -0,0 +1,17 @@
+var DeleteTask = (index) => `
+    <i id="task-${index}" class="delete-task far fa-trash-alt" style="float:right; cursor: pointer"></i>
+`
+
+
+DeleteTask.addEventListeners = (tasks) => {
+    var deleteButton = document.querySelectorAll('.delete-task')
+    deleteButton.forEach(element => {
+        element.addEventListener('click', event => {
+            event.target.parentNode.parentNode.removeChild(event.srcElement.parentNode);
```

Comment:
Two prblems with this line:
- `srcElement` is not standard, you should use `target` instead.
- You don't need to remove the node from the DOM because `renderApp` will do it.